x-pack/plugins/enterprise_search/server/lib/get_oauth_token_package_params.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENTERPRISE_SEARCH_KIBANA_COOKIE } from '../../common/constants';
+
+export function getOAuthTokenPackageParams(cookieHeaders: string | string[] | undefined) {
+  // In the future the token package will be stored in the login session. For now it's in a cookie.
+
+  // No cookie headers? No tokens
+  if (!cookieHeaders) {
+    return {};
+  }
+
+  // Take any cookie headers and split the individual cookies out, e.g. "_my_cookie=chocolateChip"
```

Comment:
Clearly my comment isn't as useful as I'd hoped, because that isn't the end result of the line being commented 😄 

It's taking something like this:
```json
[
  "_my_cookie=chocolateChip; _other_cookie=oatmeal",
  "_other_header_other_cookie=snickerdoodle",
]
```

and turning it into something like this:
```json
[
  "_my_cookie=chocolateChip",
  "_other_cookie=oatmeal",
  "_other_header_other_cookie=snickerdoodle",
]
```

Perhaps adding a comment with a little background on how cookies are structured would be helpful?

Reply:
Sorry, I assumed the comment was giving an example of the input - maybe it might help to be more specific, e.g.

```js
// '_my_cookie=chocolateChip; _other_cookie=oatmeal' becomes ['_my_cookie=chocolateChip', '_other_cookie=oatmaeal']
```

That being said, I think we're straying into territory where examples and documentation are better given in the unit tests than in comments. Could I suggest this instead then:

```suggestion
  // Take any cookie headers and split the individual cookies out. See unit tests for example inputs
```